src/common/ceph_time.h

```diff
@@ -181,6 +181,14 @@ namespace ceph {
 	return from_timespec(ts);
       }
 
+      static bool is_zero(const time_point& t) {
+	return (t == time_point::min());
```

Comment:
nit: indentation?

Reply:
or maybe just github issues with tabs/spaces?

Reply:
yeah, i just copied the tab formatting used elsewhere in the file